### ISSUE TITLE
Emoji misalignment on Tutorial Metadata

### ISSUE THREAD

**Issue by vdusart** [ethereum/ethereum-org-website#8718]
### Describe the bug

On the tutorial list page and on the tutorial page, the metadata are not aligned

### To reproduce

1. Go to [a tutorial](https://ethereum.org/en/developers/tutorials/short-abi/) or [the list of tutorials](https://ethereum.org/en/developers/tutorials/)
2. Look at the tutorial metadata (part just bellow the title)
3. See the not aligned elements (icons and texts)

### Expected behavior

All the text and icons should be aligned

### Screenshots

![image](https://user-images.githubusercontent.com/43795504/203425660-08e73e8a-82be-45d2-b52e-615568a43120.png)

### Desktop (please complete the following information)
Same

### Smartphone (please complete the following information)

Same

### Additional context

I can work on it :smile:

**Comment by vdusart** [comment]
Hey @pettinarip, I took a look at it and here is what I found for the moment.
I don't know why but the `:writing_hand:` emoji creates an empty node in the span (just bellow the `img` tag) as you can see on the image bellow
![image](https://user-images.githubusercontent.com/43795504/203667734-ebdd84e6-d54d-442e-a936-c7a19453c5f5.png)


When I delete we obtain the wanted result
![image](https://user-images.githubusercontent.com/43795504/203666993-c6012f40-92e3-4fa9-8339-724b61c827dc.png)
Please tell me if you have any idea :smile: 

# Other solution
In the worst case, if we don't find what's the problemn and we can't solve it, we can use the :memo: emoji instead of the :writing_hand: one.
It will look like that:
![image](https://user-images.githubusercontent.com/43795504/203666906-1952cd21-070d-4200-b692-a1d8b62cab81.png)
instead of:
![image](https://user-images.githubusercontent.com/43795504/203666993-c6012f40-92e3-4fa9-8339-724b61c827dc.png)

**Comment by TylerAPfledderer** [comment]
@pettinarip @vdusart  The global `img, svg, video, canvas, audio, iframe, embed, object` multi-selector has `display: block` which is causing the white space (very strange indeed this is only happening with the one emoji!).

I would advise going into the descendant selector of [Emoji](https://github.com/ethereum/ethereum-org-website/blob/cd0c9575bcffb068538568f3d3cbae76b8564021/src/components/Emoji.tsx#L21-L25) and apply `display: initial`.

```tsx
sx={{
  "& > img": {
    margin: "0 !important",
    display: "initial", // reverts back to `inline`
  },
}}
```

Question here is whether or not this should go ahead and be applied in #8717

**Comment by pettinarip** [comment]
Would be nice to add that fix on #8717 @vdusart

**Comment by pettinarip** [comment]
Great suggestion @TylerAPfledderer we ended up implementing that 💪🏼 

Thanks for the implementation & fixes @vdusart 🙏🏼 

Closing this for now.
